packages/@lwc/engine/src/framework/vm.ts

```diff
@@ -86,9 +86,9 @@ export interface UninitializedVM {
     /** Adopted Children List */
     aChildren: VNodes;
     velements: VCustomElement[];
-    cmpProps: any;
+    cmpProps: Record<string, any>;
     cmpSlots: SlotSet;
-    cmpTrack: any;
+    cmpFields: Record<string, any>;
```

Comment:
this is just a rename since the `cmpTrack` was exclusively a hash table for tracked fields, but in general, since they do not overlap, we can just generalize that for all important fields.